### ISSUE TITLE
Method of removing title parameter causes crash in IE9

### ISSUE THREAD

**Issue by lavoieaj** [remarkablemark/html-dom-parser#6]
This surfaced through my use of html-react-parser.

Console throws `SCRIPT600: Invalid target element for this operation.`

Debugger points to `doc.documentElement.innerHTML = '';` in domparser.js

https://msdn.microsoft.com/en-us/library/ms533897(v=vs.85).aspx

> The innerHTML property is read-only on the col, colGroup, frameSet, html, head, style, table, tBody, tFoot, tHead, **title**, and tr objects.
> You can change the value of the title element using the document.title property.

**Comment by remarkablemark** [comment]
Thanks for bringing this to my attention @lavoieaj

Can you help me reproduce this bug using [JSFiddle](https://jsfiddle.net/remarkablemark/1txvmjuq/)? This way I can test it in my IE. Thanks!

**Comment by lavoieaj** [comment]
The JSFiddle you provided throws the original error as well as an error related to the JSFiddle editor. My testing environment is Windows 10/IE11 running in IE9 document mode.

JSFiddle editor only appears to work in IE9 document mode when I start with a blank JSFiddle window and then paste in javascript. 

https://jsfiddle.net/nq7yusd6/1/ is the same as your example above, just easier to paste as a workaround
https://jsfiddle.net/nq7yusd6/2/ shows where I replaced `c.documentElement.innerHTML=""` with `c.title=""`

**Comment by remarkablemark** [comment]
Thanks for the additional update. I was able to reproduce the bug in a similar environment (Windows 10, IE11 running IE9 document mode).

My updated [fiddle](https://jsfiddle.net/remarkablemark/1txvmjuq/2/) should take care of the major errors found in IE9. For reference, they were coming from [`html-dom-parser/lib/domparser.js`](https://github.com/remarkablemark/html-dom-parser/blob/v0.1.1/lib/domparser.js#L48).

Feel free to review my solution before I make a pull request.

**Comment by remarkablemark** [comment]
@lavoieaj The fix is in [`html-dom-parser@0.1.2`](https://github.com/remarkablemark/html-dom-parser/tree/v0.1.2), which is included in [`html-react-parser@0.3.6`](https://github.com/remarkablemark/html-react-parser/tree/v0.3.6):
```sh
# npm
npm install html-react-parser@0.3.6 --save

# yarn
yarn upgrade html-react-parser@0.3.6
```

If you've upgraded to `react` and `react-dom` 16, then make sure to use the latest version of [`html-react-parser@0.4.0`](https://github.com/remarkablemark/html-react-parser/tree/v0.4.0):
```sh
# npm
npm install html-react-parser@0.4.0 --save

# yarn
yarn upgrade html-react-parser@0.4.0
```

Let me know if you experience any issues.
